lib/airbrake-ruby.rb

```diff
@@ -286,7 +286,9 @@ def call_notifier(notifier, method, *args, &block)
   end
 end
 
-# Notify of unhandled exceptions, if there were any.
+# Notify of unhandled exceptions, if there were any, but ignore SystemExit.
 at_exit do
-  Airbrake.notify_sync($ERROR_INFO) if $ERROR_INFO
+  if $ERROR_INFO && $ERROR_INFO.class != SystemExit
```

Comment:
I doubt there's an easy way. I'd prefer not to mess with processes. Luckily, this hook will not get any more complex.